### PR TITLE
Create initial project gov docs

### PR DESCRIPTION
**Pull request by droumis**

### CODE REVIEW

File: doc/governance/project-docs/MEMBERS.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+For member policy, see the description at the top of [HoloViz/HoloViz - MEMBERS.md](https://github.com/holoviz/holoviz/blob/colorcet-gov/doc/governance/project-docs/MEMBERS.md).
+
+
+The Colorcet Project’s equivalently named documents take precedence over any external materials referenced within this linked document above.
+
+
+| **NAME** | **Role** | **GitHub Handle** |
+| --- | --- | --- |
+| James Bednar | Project Director | [jbednar](https://github.com/jbednar) |
```
[the remark this answers is not in the document]
Comment (maximlt) [reply doc/governance/project-docs/MEMBERS.md:12]: I guess my contributions are mostly CI related. Let's discuss that during one of our meetings, there's maybe someone interested in colormaps and happy to take over some of the colormappy maintenance work here.

[the remark this answers is not in the document]
Comment (ianthomas23) [reply doc/governance/project-docs/MEMBERS.md:12]: I am happy to be a maintainer here, as a regular user of colormaps.

[the remark this answers is not in the document]
Comment (droumis) [reply doc/governance/project-docs/MEMBERS.md:12]: I'm happy to be considered to be added as a maintainer after I've made some substantial contributions to the docs. I guess this brings up a question that should be discussed for all of HoloViz - whether someone contributing primarily to the docs should be allowed to vote (i.e. get maintainer status). I can see an argument either way. Maybe they could vote on docs-related issues? I'm not sure what the answer is here and I'll have to take a look at how other projects handle this. I guess Timo (Bokeh) is an example of a docs contributor with core voting rights (I think)

[the remark this answers is not in the document]
Comment (ianthomas23) [reply doc/governance/project-docs/MEMBERS.md:12]: > I guess Timo (Bokeh) is an example of a docs contributor with core voting rights (I think)

Correct. Bokeh is lucky to have an eclectic group of core maintainers.

Diversity of contributors and maintainers is a really good goal to aim for.

[the remark this answers is not in the document]
Comment (maximlt) [reply doc/governance/project-docs/MEMBERS.md:12]: Very happy not to be a maintainer of colorcet and just an occasional contributor :)

I agree with Ian, diversity is good!

[the remark this answers is not in the document]
Comment (jbednar) [reply doc/governance/project-docs/MEMBERS.md:12]: I think the key aspect that determines eligibility is simply "feeling of ownership". Someone who contributes to the docs or CI but feels no attachment to this particular project wouldn't be eligible. Someone who submits all the same PRs but does feel ownership of this particular project, i.e. a sense of wanting to improve and help this particular project succeed into the future, is eligible for full voting rights on all matters, regardless of the nature or type of contribution. People can always abstain from making decisions on something they don't have opinions about. 

So, let's add Ian as a maintainer, who just expressed ownership, while Maxime and Demetris expressed what sounded to me like not (yet) expressing ownership of this particular project. That's my vote!

**Review by ianthomas23 — APPROVED** [review]

**Comment by droumis** [comment]
Is this ready to merge?

**Comment by ianthomas23** [comment]
I believe so.

**Pull request merged**